docs/source/fundamentals.md

```diff
@@ -101,29 +99,155 @@ Once a certain degree of interoperability has been reached, other interesting co
+	but complex to maintain/read by a human.
 
-  Here is an example of what that could look like:
+	The following representation should be in one line (it has been split for readability)
+  ```xml
+    <http://dbpedia.org/resource/The_Lord_of_the_Rings>
+      <http://dbpedia.org/ontology/author>
+      <http://dbpedia.org/resource/J._R._R._Tolkien> .
+  ```
+	
+- [JSON-LD](https://www.w3.org/TR/json-ld/): uses the commonly accepted web data scheme for serialising RDF triples.
+	Easier than XML for humans, JSON has standard libraries in practically all programming languages.
+
+	The example in JSON is:
+  ```json
+    {"@id": "http://dbpedia.org/resource/The_Lord_of_the_Rings",
+      "http://dbpedia.org/property/author": 
+        [{"@id": "http://dbpedia.org/resource/J._R._R._Tolkien"}]
+      }
+  ```
+SimPhoNy supports all the previous formats (plus a simpler custom YAML) as inputs in the ontology installation.
+
+#### SPARQL
+[SPARQL](https://www.w3.org/TR/sparql11-overview/) (recursively SPARQL Protocol and RDF Query Language) is the most common query language for RDF.
+Queries are graph patterns (similar to the triples of Turtle) with variables for the parts of the pattern that make up the result.
+
+Variables start with the identifier `?` and represent concrete values that will be matched in the query process.
+They can appear in multiple locations in the patterns and those present in the 
+`SELECT` clause will be returned as the query result.
+
+The query for the author of _The Lord of the Rings_ from our sample triples in SPARQL is:
+  ```sparql
+    PREFIX dbo: <http://dbpedia.org/ontology/>
+    PREFIX dbr: <http://dbpedia.org/resource/>
+    SELECT ?person WHERE {
+        dbr:The_Lord_of_the_Rings  dbo:author  ?person .
+    }
+  ```
 
-  <iframe src="./_static/videos/coupling_and_linking.mp4" frameborder="0" allowfullscreen="true">
-  </iframe>
+The SPARQL query language offers multiple types of result sets and clauses, most of which won't be used for this Master's thesis.
+One which should be mentioned is the `FILTER` keyword.
+This will limit the result to those that evaluate `true` to the expression inside the brackets.
+For instance (omitting the prefix declaration for simplicity):
+
+  ```sparql
+    SELECT ?character WHERE {
+        ?character dbp:affiliation dbr:The_Lord_of_the_Rings .
+        ?character dbo:age ?age .
+        FILTER(?age >= 100)
+    } 
+  ```
+The previous query would return the characters from the book series with an  age higher or equal to 100.
+(Note that while the query is correct, the result is empty, as such information is not stored on DBpedia).
 
-  Furthermore, a truly interoperable platform would enable users to store and 
-  access data in databases or other repositories of information.
+For a very interesting and comprehensive introduction into RDF and SPARQL, see [[Hitzler, 2009]](http://dx.doi.org/10.1201/9781420090512).
```

Comment:
Perhaps "reasoning" should be mentioned as well, as some people were got confused by this term